basis/common-token/common-token-api/src/main/java/com/softgroup/common/token/api/TokenService.java

```diff
@@ -0,0 +1,13 @@
+package com.softgroup.common.token.api;
+
+/**
+ * Created by Виктор on 20.03.2017.
+ */
+public interface TokenService {
+
+    String generateToken(String userId, Long timeleft, String typeToken);
```

Comment:
"timeleft" is neccessary, because I need to put expiration time to token.  I will add deviceId to token.